src/js/app.js

```diff
@@ -39,3 +39,10 @@ if (styleGuide) {
     console.log('Style guide module loaded.');
   });
 }
+
+const tagsInput = document.querySelector('[data-tags-input]');
+if (tagsInput) {
+  import(/* webpackChunkName: "tags-input" */ './tags-input.js').then(() => {
+    console.log('Tags input module loaded.');
```

Comment:
I think it's time to get rid of those logs after module loading in production so the end user doesn't see them.